$NetBSD: patch-ab,v 1.6 2011/07/18 12:02:02 tron Exp $

Fix broken shared library version number. Create with "autoreconf" after
applying "patch-aa". Please look here for more details:

https://bugzilla.gnome.org/show_bug.cgi?id=654767

--- configure.orig	2011-06-13 17:34:43.000000000 +0100
+++ configure	2011-07-18 12:44:14.000000000 +0100
@@ -2961,7 +2961,7 @@
 ATK_VERSION=2.0.1
 ATK_API_VERSION=1.0
 ATK_INTERFACE_AGE=1
-ATK_BINARY_AGE=11
+ATK_BINARY_AGE=20011
 
 
 
@@ -2971,7 +2971,7 @@
 
 
 
-LT_VERSION_INFO=10:1:10
+LT_VERSION_INFO=20010:1:20010
 LT_CURRENT_MINUS_AGE=0
 
 
